site/en/docs/extensions/mv3/architecture-overview/index.md

```diff
@@ -66,6 +66,9 @@ An extension can have different HTML pages depending on the design. All extensio
 [The options page][doc-options]
 : The options page (`options.html`) provides a way for users to customize an extension, such as choosing which sites the extension will run on. Users can access the options page in several ways as described in [Finding the options page][doc-options-view].
 
+[Side panels][api-sidepanel]
+: Extensions can also use side panels (`sidepanel.html`) to provide additional functionality alongside the main content of a site. Users can open side panels by navigating to Chrome’s built-in sidebar menu or by clicking the extension toolbar icon. Side panels can be displayed only on specific sites.
```

Comment:
Tweaked this a little to make it clear that there's a single side panel that may contain different content, and also that it isn't limited to specific sites unless you want it to be :)

```suggestion
: Extensions can also provide UI to appear in the side panel (`sidepanel.html`) to provide additional functionality alongside the main content of a site. Users can open these by opening the side panel in Chrome or by clicking the extension toolbar icon. Side panels can be configured to only be displayed on specific sites.
```

Reply:
Good point! 👍